src/parser/priest/holy/modules/core/EchoOfLightMastery.tsx

```diff
@@ -105,16 +110,18 @@ class EchoOfLightMastery extends Analyzer {
     }
   }
 
-  handleEolTick(event) {
+  handleEolTick(event: any) {
     const targetId = event.targetID;
 
     // As far as I can tell, this happens when the combat log is out of order. You shouldn't receive a tick of EoL without a target having a buff apply event.
     if (!this.targetMasteryPool[targetId]) {
+      // @ts-ignore
```

Comment:
What was the error here so we can resolve it (and L124, and L208)

Reply:
The combatant module isn't typed yet, so this is throwing an error.

TS7053: Element implicitly has an 'any' type because expression of type 'number' can't be used to index type '{}'.   No index signature with a parameter of type 'number' was found on type '{}'.